docs/outlook/autolaunch.md

```diff
@@ -327,7 +327,7 @@ Add-in launch-event handlers are expected to be short-running, lightweight, and
 
 If the user has multiple add-ins that subscribed to the same event, the Outlook platform launches the add-ins in no particular order. Currently, only five event-based add-ins can be actively running.
 
-The user can switch or navigate away from the current mail item where the add-in started running. The add-in that was launched will finish its operation in the background.
+In Outlook on Windows and on the web, the user must remain on the current mail item where the add-in was activated for it to complete running. Navigating away from the current item (for example, switching to another compose window or tab) terminates the add-in operation. The add-in also ceases operation when the user sends the message or appointment they're composing.
```

Comment:
What's the behavior for Mac? Or is this for a certain subset of events?

Reply:
Thanks for catching that. This applies to all supported clients, including Mac, and also applies to all events.